src/constants/Blacklist.js

```diff
@@ -51,7 +51,7 @@ const swearRegex = new RegExp(
             str +
             swear
                 // This function makes the swear allow spaces inbetween letters
-                .replace(/(?:(?:\(.+?\)(?:\{.+?\})?)|[a-z=.-])(?:\+|\?|\*|\$)?/g, '$&\\s*') +
+                .replace(/(?:(\(.+?\)(?:{.+?})?)|[a-z=.-])([+?*$])?/g, '$&\\s*') +
```

Comment:
Why are we removing the `?:` here?

Reply:
bc it doesn't do anything. it is just redundant

Reply:
It tells regex not to store this which actually is more performant.

Reply:
as if such a small change really matters performance wise. if you want to talk about performance, look at the rest of the code

Reply:
Ok fine